linuxdeploy-plugin-gtk.sh

```diff
@@ -226,11 +226,20 @@ mkdir -p "$HOOKSDIR"
 cat > "$HOOKFILE" <<\EOF
 #! /usr/bin/env bash
 
+FDO_COLOR_SCHEME=$(dbus-send --session --dest=org.freedesktop.portal.Desktop --type=method_call --print-reply --reply-timeout=1000 /org/freedesktop/portal/desktop org.freedesktop.portal.Settings.Read 'string:org.freedesktop.appearance' 'string:color-scheme' 2>/dev/null | tail -n1 | cut -b35- | cut -d' ' -f2)
+if [[ -n "$FDO_COLOR_SCHEME" ]]; then
+case "$FDO_COLOR_SCHEME" in
```

Comment:
I would prefer to use the same `case` statement for both. Set a variable (like `$color_scheme`) containing value of `$FDO_COLOR_SCHEME` or `$(gsettings get org.gnome.desktop.interface color-scheme 2> /dev/null)` then:
```sh
case "$color_scheme" in
    "1"|"'prefer-dark'")  GTK_THEME_VARIANT="dark";;
    "2"|"'prefer-light'") GTK_THEME_VARIANT="light";;
    *)                    GTK_THEME_VARIANT="light";;
esac
```